src/utils/particle/drawer.ts

```diff
@@ -174,12 +188,12 @@ export class Drawer {
         ctx.drawImage(img_obj, particle.position.x - radius, particle.position.y - radius, radius * 2, radius * 2 / ratio);
     }
 
-    public static subDrawShape(ctx: CanvasRenderingContext2D, startX: number, startY: number, sideLength: number, sideCountNumerator: number, sideCountDenominator: number) {
+    private static subDrawShape(ctx: CanvasRenderingContext2D, startX: number, startY: number, sideLength: number, sideCountNumerator: number, sideCountDenominator: number) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)